apps/emqx_authz/src/emqx_authz_schema.erl

```diff
@@ -153,11 +152,6 @@ mongo_common_fields() ->
                  default => true}}
     ].
 
-validations() ->
```

Comment:
`emqx_authz_schema` is never used directly, so these validations are never called.